### PR TITLE
Fixes to dev/reset_dbs.py, fix scipy.misc.comb imports and enable wbia_cnn plugin

### PR DESCRIPTION
**Pull request by karenc**
- Change wbia to ibeis in hsdb convert success flag filename

  We previously changed the filename to `_hsdb_to_wbia_convert_success`.
  This became a problem when running `dev/reset_dbs.py`:
  
  ```
  Traceback (most recent call last):
    File "dev/reset_dbs.py", line 14, in <module>
      reset_testdbs.reset_testdbs()
    File "/wbia/wildbook-ia/wbia/tests/reset_testdbs.py", line 181, in reset_testdbs
      wbia.ensure_pz_mtest()
    File "/wbia/wildbook-ia/wbia/init/sysres.py", line 425, in ensure_pz_mtest
      ibs = wbia.opendb('PZ_MTEST')
    File "/wbia/wildbook-ia/wbia/main_module.py", line 601, in opendb
      ibs = _init_wbia(dbdir, verbose=verbose, use_cache=use_cache, web=web, **kwargs)
    File "/wbia/wildbook-ia/wbia/main_module.py", line 113, in _init_wbia
      force_serial=force_serial,
    File "/wbia/wildbook-ia/wbia/control/IBEISControl.py", line 254, in request_IBEISController
      dbdir, ensure=ensure, wbaddr=wbaddr, verbose=verbose
    File "/wbia/wildbook-ia/wbia/dbio/ingest_hsdb.py", line 337, in convert_hsdb_to_wbia
      assert len(ibs.get_valid_gids()) == 0, 'target database is not empty'
  AssertionError: target database is not empty
  ```
  
  The code couldn't detect that the database doesn't need to be converted
  because the filename changed, so change it back to
  `_hsdb_to_ibeis_convert_success`.

- Fix flake8 errors in unstable/old_dbinfo_dan.py

  ```
  wbia/unstable/old_dbinfo_dan.py:311:5: F841 local variable 'is_hyrbid' is assigned to but never used
  ```
  
  flake8 is not recognizing the `# NOQA` at the end of the list, instead
  it needs to go on the first line of the list, after `[`.

- Import comb from scipy.special instead of scipy.misc

  `comb` has been moved from `scipy.misc` to `scipy.special`:
  
  ```
  AttributeError: module 'scipy.misc' has no attribute 'comb'
  ```

- Rename ibeis_cnn references to wbia_cnn

  ```
  git grep -l ibeis_cnn ':^docs' | xargs sed -i 's/ibeis_cnn/wbia_cnn/g'
  ```

- Enable wbia_cnn plugin in control/IBEISControl.py

  When running doctests:
  
  ```
  ...
    File "/wbia/wildbook-ia/wbia/core_annots.py", line 823, in cnn_probchips
      mask_gen = ibs.generate_species_background_mask(inputchip_fpaths, species)
  AttributeError: 'IBEISController' object has no attribute 'generate_species_background_mask'
  DOCTEST REPRODUCTION
  CommandLine:
      python -m xdoctest /wbia/wildbook-ia/wbia/web/graph_server.py GraphActor:1
  ```
  
  `generate_species_background_mask` is defined in the wbia cnn plugin.  It needs
  to be added to `AUTOLOAD_PLUGIN_MODNAMES` in
  `wildbook-ia/wbia/control/IBEISControl.py` for the method to be added.

### CODE REVIEW

File: dev/docker/Dockerfile [hunk 1]
```diff
@@ -72,7 +72,7 @@ RUN mkdir -p /root/.ssh
 # Add deploy keys and SSH keys for Github with this container
 ADD dev/keys/deploy-wbia /root/.ssh/deploy-wbia
 
-ADD dev/keys/deploy-ibeis_cnn /root/.ssh/deploy-ibeis_cnn
+ADD dev/keys/deploy-wbia_cnn /root/.ssh/deploy-wbia_cnn
```
Comment (bluemellophone) [inline dev/docker/Dockerfile:75]: We can remove all of the keys, we aren't using them anymore since we are using open source versions of these repos.

Comment (karenc) [reply dev/docker/Dockerfile:75]: Should I just remove the `dev/docker` stuff or may be leave those files unchanged?  The ones I worked on are in `devops/` and I'm not sure what to do with the docker files in `dev/docker`.

Comment (bluemellophone) [reply dev/docker/Dockerfile:75]: I understand, I forgot we had an old Dockerfile in dev.  Yes, just delete the entire `dev/docker` folder.

Comment (karenc) [reply dev/docker/Dockerfile]: Ok deleted :+1:

**Pull request merged**